util/linux_install.sh

```diff
@@ -126,6 +126,39 @@ elif grep ID /etc/os-release | grep -qE "opensuse|tumbleweed"; then
 		wget \
 		zip
 
+elif grep ID /etc/os-release | grep -q slackware; then
+	echo "You will need the following packages from slackbuilds.org:"
+	echo " avr-binutils"
+	echo " avr-gcc"
+	echo " avr-libc"
+	echo " avrdude"
+	echo " dfu-programmer"
+	echo " dfu-util"
+	echo " arm-binutils"
+	echo " arm-gcc"
+	echo " newlib"
+	echo ""
+	echo -n "Would you like to install them with sudo and sboinstall (y/N)? "
+	old_stty_cfg=$(stty -g)
+	stty raw -echo
+	answer=$( while ! head -c 1 | grep -i '[ny]' ;do true ;done )
+	stty $old_stty_cfg
```

Comment:
The reason for the confirmation and message is because sudo is not configured by default, also users are not added by default to sudo group and sboinstall (from sbotools) is not an official package manager, so it is very likely that for most users the installation command will fail. Is that ok?

Reply:
Alright, that's totally fine.